# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linux-header/ppc64-asm-types.h.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./include/asm-ppc64/types.h.vanilla	2006-03-29 08:14:17.000000000 +0200
+++ ./include/asm-ppc64/types.h	2006-03-29 08:14:51.000000000 +0200
@@ -16,7 +16,7 @@
  * 2 of the License, or (at your option) any later version.
  */
 
-typedef unsigned int umode_t;
+//ttypedef unsigned int umode_t;
 
 /*
  * __xx is ok: it doesn't pollute the POSIX namespace. Use these in the
